rts/Rendering/Env/CubeMapHandler.cpp

```diff
@@ -62,8 +62,8 @@ bool CubeMapHandler::Init() {
 		glBindTexture(GL_TEXTURE_CUBE_MAP, envReflectionTexID);
 		glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_MIN_FILTER, generateMipMaps? GL_LINEAR_MIPMAP_LINEAR: GL_LINEAR);
 		glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_MAG_FILTER,                                           GL_LINEAR); // magnification doesn't use mips
-		glTexParameteri(GL_TEXTURE_CUBE_MAP, GL_TEXTURE_WRAP_S, globalRendering->supportSeamlessCubeMaps? GL_MIRRORED_REPEAT: GL_CLAMP_TO_EDGE);
```

Comment:
Why?

Reply:
Don't remember exact reason. Possibly got killed accidentally.
Shouldn't be relevant though. Face textures occupy each face to fit exactly, so wrapping shouldn't matter.